src/ddestats.jl

```diff
@@ -0,0 +1,34 @@
+mutable struct DDEStats
+  nf::Int
+  nf2::Int
+  nw::Int
+  nsolve::Int
+  njacs::Int
+  nnonliniter::Int
+  nnonlinconvfail::Int
+  nfpiter::Int
+  nfpconvfail::Int
+  ncondition::Int
+  naccept::Int
+  nreject::Int
+  maxeig::Float64
```

Comment:
We should double check whether this works with AD. IIRC there is an `internalnorm` before the save here so we're good.

Reply:
It's copied from DiffEqBase: https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/destats.jl

Reply:
okay awesome then it must work.